src/frontend/src/components/Layout/Layout.js

```diff
@@ -18,32 +16,64 @@ const items = [
   { link: '#', text: 'About' },
 ];
 
+const isScrollBottom = () =>
+  window.innerHeight + window.scrollY >= document.documentElement.scrollHeight;
+
 class Layout extends Component {
   constructor(props) {
     super(props);
     this.state = {
       sideDrawerOpen: false,
       scrolled: false,
+      numPages: 1,
+      posts: [],
     };
     this.drawerToggle = this.drawerToggle.bind(this);
     this.backdropClick = this.backdropClick.bind(this);
   }
 
   componentDidMount() {
+    this.getPosts(this.state.numPages);
     window.addEventListener('scroll', () => {
       const isTop = window.scrollY > 320;
       if (isTop) this.setState({ scrolled: true });
       else this.setState({ scrolled: false });
+
+      if (isScrollBottom()) {
+        this.setState(prevState => ({ numPages: prevState.numPages + 1 }));
+        this.getPosts(this.state.numPages);
+      }
     });
   }
 
+  async getPosts(pageNum = 1) {
+    let postsData = [];
+    try {
+      const res = await fetch(`http://dev.telescope.cdot.systems/posts?page=${pageNum}`);
```

Comment:
We should pull the API base URL from an env variable.  Let's do that in a follow-up.